### ISSUE TITLE
Disable Cmd + Click

### ISSUE THREAD

**Issue by yeefun** [Viijay-Kr/react-ts-css#166]
Type: <b>Bug</b>

It disables Cmd + Click

Extension version: 3.2.1
VS Code version: Code 1.91.0 (ea1445cc7016315d0f5728f8e8b12a45dc0a7286, 2024-07-01T18:53:23.353Z)
OS version: Darwin arm64 23.5.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Pro (10 x 2400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|5, 5, 5|
|Memory (System)|16.00GB (1.53GB free)|
|Process Argv|--crash-reporter-id 76d51adc-9dc0-4aac-8947-cdcdeb20404f|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscoreces:30445986
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
welcomedialog:30910333
pythonnoceb:30805159
asynctok:30898717
pythonregdiag2:30936856
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
accentitlementsc:30995553
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
jchc7451:31067544
chatpanelt:31048053
dsvsc021:30996838
da93g388:31013173
pythoncenvpt:31062603
a69g1124:31058053
dvdeprecation:31068756
dwnewjupytercf:31046870
newcmakeconfigv2:31071590
legacy_priority:31082724

```

</details>

**Comment by RoyiNamir** [comment]
Yes. You've fucked up the whole ctrl+ mouse click with the latest version.
plz do tests before you publish a package

**Comment by Viijay-Kr** [comment]
@RoyiNamir I m sorry that you are experiencing the issue this issue is being tracked by #163

**Comment by Viijay-Kr** [comment]
@RoyiNamir @yeefun It should be fixed by v3.2.3 . Please update the extension. Apologies for the inconvenience
